frontend/vue/components/CodeExercise/ExerciseActionsBar.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+  <div class="exercise-actions-bar">
+    <button
+      v-if="runEnabled && !isRunning"
+      class="exercise-actions-bar__button exercise-actions-bar__button_run"
+      @click="run()"
+    >
+      Run
```

Comment:
Similar comment regarding translations, something like:

```
{{ $translate('Run') }}
```

then make sure the strings are defined in `strings.yaml` file